static/app/utils/analytics/projectCreationAnalyticsEvents.tsx

```diff
@@ -1,4 +1,16 @@
 export type ProjectCreationEventParameters = {
+  'project_creation.back_button_clicked': {};
```

Comment:
already took the opportunity and add this analytics code so we can track how users are using the back button in the project creation flow. Please let me know if it is ok to have it here or if you prefer it in a separate PR